app/Http/Controllers/EmailVerifyController.php

```diff
@@ -22,14 +22,19 @@ public function __invoke(EmailVerificationRequest $request): RedirectResponse
 
     public function create(): Response
     {
-        return Inertia::render("Auth/Verify-Email");
+        return Inertia::render("Auth/VerifyEmail");
     }
 
     public function send(Request $request): RedirectResponse
     {
-        $request->user()->sendEmailVerificationNotification();
+        if (auth()->check()) {
+            $request->user()->sendEmailVerificationNotification();
+
+            return back()
+                ->with("status", "Wiadomość z linkiem aktywacyjnym została wysłana na Twój adres e-mail!");
+        }
 
         return back()
-            ->with("message", "Wiadomość z linkiem aktywacyjnym została wysłana na Twój adres e-mail!");
+            ->with("status", "Wiadomość z linkiem aktywacyjnym została wysłana na Twój adres e-mail!");
```

Comment:
I believe we can simplify logic of this method - something like

public function send(Request $request): RedirectResponse
{
    if (auth()->check()) {
        $request->user()->sendEmailVerificationNotification();
    }

    return back()
        ->with("status", "Wiadomość z linkiem aktywacyjnym została wysłana na Twój adres e-mail!");
}

Reply:
Yes, you're right.